tests/bench.sh

```diff
@@ -0,0 +1,14 @@
+#!/bin/bash
+function locate {
+     sed -n "$1"p "$3"
```

Comment:
I guess this prints the line on which Merlin is called ?
Where will this printing be visible ? In some logs ?

Reply:
yes exactly, and we can see this print in the benchmark result "as you can see in this link http://autumn.ocamllabs.io:8081/job/2022-06-23/125306-ocluster-build-9d17b9#L283-L296" and on the terminal.